tests/yast2_cmd/yast_ftp_server.pm

```diff
@@ -121,9 +121,9 @@ sub run {
     assert_script_run 'yast ftp-server welcome_message set_message="hello everybody" ';
 
     #show, displays FTP server settings.
-    my @pattern = ('Start-Up:FTP daemon needs manual starting',
+    my @pattern = ('Start-Up:FTP daemon',    # the rest of sentence differs between versions (xinetd, socket...)
         'hello everybody',
-        'Anonymous and Authenticated Users');
```

Comment:
This still causes trouble on sle12, but workaround has to go. AFAIS, we just check random parts of output anyway.